ports/pcapplusplus/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "pcapplusplus",
-  "version": "22.11",
+  "version": "23.09",
```

No, there can't be leading zeros in version scheme `version`, even if vcpkg reviewer suggest it.
```suggestion
  "version": "23.9",
```

---

Can I change it to version-string ?

---

`version-string` doesn't allow for `version>=`.